pkgs/by-name/kb/kbd/package.nix

```diff
@@ -13,10 +13,18 @@
   bzip2,
   xz,
   zstd,
+  bash,
   gitUpdater,
+  # This might be worthwhile to disable if the store lives on a compressed filesystem.
+  withCompression ? true,
+  # Allow disabling the obsolete unicode_start/unicode_stop utilities,
+  # which unfortunately pull in bash into the closure.
+  withObsolete ? true,
 }:
 
 stdenv.mkDerivation rec {
+  strictDeps = true;
```

Comment:
I think with strictDeps, we might have to explicitly add gzip to the inputs when compression is enabled

Reply:
They are picked up and then it still doesn't work. Not it's fixed.